templates/index.html

```diff
@@ -1,27 +1,968 @@
+                    </select>
+                  </div>
+                </div>
+              </div>
+              <footer class="mdc-dialog__footer">
+                <button
+                  type="button"
+                  class="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--cancel"
+                >
+                  Cancel
+                </button>
+                <button
+                  type="button"
+                  class="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--accept mdc-dialog__action"
+                >
+                  Add
+                </button>
+              </footer>
+            </div>
+            <div class="mdc-dialog__backdrop"></div>
+          </aside>
+        </div>
+      </main>
+    </div>
+
+    <div class="crypto-menu-switches">
+      <div class="crypto-menu-switches--handle">
+        <i class="material-icons">settings</i>
+      </div>
+      <!-- RTL switch -->
+      <div class="switch-wrapper mdc-theme--on-primary">
+        <div class="mdc-switch crypto-rtl-switch-selector">
+          <div class="mdc-switch__track"></div>
+          <div class="mdc-switch__thumb-underlay">
+            <div class="mdc-switch__thumb">
+              <input
+                type="checkbox"
+                id="rtl-switch"
+                class="mdc-switch__native-control"
+                role="switch"
+                aria-checked="false"
+              />
+            </div>
+          </div>
+        </div>
+        <label for="rtl-switch">RTL</label>
+      </div>
+      <!-- theme switch -->
+      <div class="switch-wrapper mdc-theme--on-primary">
+        <div class="mdc-switch crypto-theme-switch-selector">
+          <div class="mdc-switch__track"></div>
+          <div class="mdc-switch__thumb-underlay">
+            <div class="mdc-switch__thumb">
+              <input
+                type="checkbox"
+                id="crypto-theme-switch"
+                class="mdc-switch__native-control"
+                role="switch"
+                aria-checked="false"
+              />
+            </div>
+          </div>
+        </div>
+        <label for="crypto-theme-switch">Switch theme</label>
+      </div>
+    </div>
+
+    <!-- initialise material components js -->
+    <script src="https://code.jquery.com/jquery-1.9.1.min.js"></script>
+    <script src="{{ url_for('static', path='/assets/js/material-components-web.min.js') }}"></script>
+    <script>
+      mdc.autoInit();
+    </script>
+    <!-- initialise marquee js -->
+    <script src="{{ url_for('static', path='/assets/js/marquee3k.js') }}"></script>
+    <script>
+      Marquee3k.init();
+    </script>
+    <!-- layout js -->
+    <script
+      data-pace-options='{ "ajax": false, "restartOnRequestAfter": false }'
+      src="{{ url_for('static', path='/assets/js/pace.min.js') }}"
+    ></script>
+    <script src="{{ url_for('static', path='/assets/js/apexcharts.min.js') }}"></script>
```

Comment:
Can we remove `/assets/js/apexcharts.min.js` and instead use the [cdn](https://apexcharts.com/docs/installation/).
```html
<script src="https://cdn.jsdelivr.net/npm/apexcharts"></script>
```